### PR TITLE
sunny-famnit:0.1.0

### PR DESCRIPTION
**Pull request by Tiggax**
I am submitting
- [x] a new package
- [ ] an update for a package

This is a thesis template for University of Primorska, 
Faculty of Mathematics, Natural Sciences and Information Technologies

Description: Explain what the package does and why it's useful.

It is a thesis template for fast and easy final work project.

I have read and followed the submission guidelines and, in particular, I
- [x] selected a name that isn't the most obvious or canonical name for what the package does
- [x] added a `typst.toml` file with all required keys
- [x] added a `README.md` with documentation for my package
- [x] have chosen a license and added a `LICENSE` file or linked one in my `README.md`
- [x] tested my package locally on my system and it worked
- [x] `exclude`d PDFs or README images, if any, but not the LICENSE

- [x] ensured that my package is licensed such that users can use and distribute the contents of its template directory without restriction, after modifying them through normal use.

### CODE REVIEW

**Comment by Tiggax** [comment]
Follow-up question:
- Is `famnit` as a template package too ambiguous?

File: packages/preview/famnit/0.1.0/template/main.typ [hunk 1]
```diff
@@ -0,0 +1,49 @@
+#import "@preview/famnit:0.1.0": project
+
+#show project.with(
```
Comment (laurmaedje) [inline packages/preview/famnit/0.1.0/template/main.typ:3]: There are a few syntax errors in this file.

**Review by laurmaedje — COMMENTED** [review]
The name `famnit` is indeed not permissible as it is exactly the name of the faculty. We want to have a level playing field where another person from FAMNIT could also submit a template. For templates, we recommend to combine a unique adjective with the descriptive part. See the [naming section of the submission guidelines](https://github.com/typst/packages?tab=readme-ov-file#submission-guidelines) for details.

File: packages/preview/famnit/0.1.0/template/main.typ (hunk 1 above)
Comment (Tiggax) [reply packages/preview/famnit/0.1.0/template/main.typ]: Forgot the commas :sweat_smile: . I fixed them

**Comment by Tiggax** [comment]
I've some problems I missed the first time. 
And I also updated the template to use the new context.

I renamed the package to be `sunny-famnit`. I hope this is a good enough name. :sweat_smile: 

If there aren't any more problems i think it should be good to go :smile:

**Pull request merged**